website/content/en/docs/component-guides/katib.md

```diff
@@ -6,197 +6,265 @@ weight = 20
 
 ## AWS Access from Katib
```

Comment:
same comments as pipeline guide for these initial content

Reply:
done